src/qt/paymentserver.h

```diff
@@ -50,15 +43,9 @@ QT_BEGIN_NAMESPACE
 class QApplication;
 class QByteArray;
 class QLocalServer;
-class QNetworkAccessManager;
-class QNetworkReply;
-class QSslError;
 class QUrl;
 QT_END_NAMESPACE
 
-// BIP70 max payment request size in bytes (DoS protection)
-static const qint64 BIP70_MAX_PAYMENTREQUEST_SIZE = 50000;
-
 class PaymentServer : public QObject
```

Comment:
I imagine the necessary scraps can probably move out of here and we can nuke PaymentServer?

Reply:
Yes, that, and removing the network module from the Qt build can all be done as followups.